cmd/neofs-cli/modules/session/create.go

```diff
@@ -65,7 +68,15 @@ func createSession(cmd *cobra.Command, _ []string) {
 	if lfArg, _ := cmd.Flags().GetUint64(commonflags.Lifetime); lfArg != 0 {
 		lifetime = lfArg
 	}
-
+	exp, _ := cmd.Flags().GetUint64(commonflags.ExpireAt)
+	if exp != 0 {
+		var netInfoPrm internalclient.NetworkInfoPrm
+		netInfoPrm.SetClient(c)
+		ni, err := internalclient.NetworkInfo(ctx, netInfoPrm)
+		common.ExitOnErr(cmd, "can't get current epoch: %w", err)
+		currEpoch := ni.NetworkInfo().CurrentEpoch()
+		lifetime = exp - currEpoch
```

Comment:
i think i would be better to change `CreateSession` definition cause you do `NetworkInfo` here to do `NetworkInfo` one more time inside `CreateSession` later

or at least check that `exp > currEpoch`